src/app/features/fss-search/fss-search-results/fss-search-results.component.ts

```diff
@@ -108,6 +108,10 @@ export class FssSearchResultsComponent implements OnChanges {
       }
     }
   }
+
+  DownloadAll(){
```

Comment:
Please change to camel-case `downloadAll()` to match other identifiers

Reply:
Modified as per review comment